qemu/tests/multi_nics_stress.py

```diff
@@ -62,6 +62,7 @@ def run(test, params, env):
     shell_port = params.get("shell_port")
     os_type = params.get("os_type")
     shell_prompt = params.get("shell_prompt", r"^root@.*[\#\$]\s*$|#")
+    disable_firewall = params.get("disable_firewall", "")
```

Comment:
Suggest that give a default value if call pramas.get() , or call params["disable_firewall"] if the parameter "disable_firewall" must be given in cfg.
Thanks.

Reply:
Ｉ have update the code, please help review. thx
[root@dell-per730-06 staf-kvm-devel]# /usr/bin/avocado run --journal --vt-config /var/lib/avocado/data/avocado-vt/backends/qemu/cfg/tests-example.cfg
JOB ID     : ae392cfb50a8ad78b6ff98cb04696852025729b5
JOB LOG    : /root/avocado/job-results/job-2018-07-06T02.47-ae392cf/job.log
 (1/1) Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.multi_nics_stress: PASS (596.23 s)

Reply:
@quanwenli Now the default value is none, it should be meaningful. It's better to use params["disable_firewall"], this will ensure to disable firewall.

Reply:
@quanwenli if i understand your purpose right, you're trying to avoid windows guest? if the guest is using windows system, then we do not need to stop firewall, so the default value is none.

Reply:
@xiywang , May I know why does not to disable windows's firewall but with linux firewall?

Reply:
@quanwenli I asked you why the default value is none? Are you considering the windows guests? Because of the command of "service iptables stop; iptables -F; systemctl stop firewalld.service" does not suitable for windows guest?
If you're thinking what I am thinking, to avoid disable firewall on windows guest, then I'll suggest you to put "disable_firewall" in cfg file as "disable_firewall = """ for windows guest and use params["disable_firewall"] here.
Otherwise, the cmd for windows to disable firewall is "netsh advfirewall set allprofiles state off".

Reply:
Updated, please help review.